integration-tests/ledger-signer.spec.ts

```diff
@@ -185,5 +188,88 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
         done();
       });
     })
+
+    describe('Verify that use of a ledger device works with bip32', () => {
+      jest.setTimeout(60000);
+      it('Verify that the pk is correct', async (done) => {
+        const signer = new LedgerSigner(
+          transport,
+          "44'/1729'/1'/0'",
+          false,
+          DerivationType.BIP32_ED25519
+        )
+        const Tezos = new TezosToolkit(rpc);
+        Tezos.setSignerProvider(signer);
+        const pk = await Tezos.signer.publicKey();
+        expect(pk).toEqual('edpkujVjFVJtb9Z1D7jpSpPMrKzdTRZSRT8E3L26T42vvA6VSv7jND');
+        done();
+      })
+    })
+
+    describe('Verify that use of a ledger device works with bip32', () => {
+      jest.setTimeout(60000);
+      it('Verify that the pk and pkh is correct', async (done) => {
```

Comment:
This test and the precedent ('Verify that the pk is correct') could be combined into one test.